toolkit/tools/imagecustomizer/container/Dockerfile.mic-container

```diff
@@ -2,6 +2,6 @@ FROM mcr.microsoft.com/azurelinux/base/core:3.0
 RUN tdnf update -y && \
    tdnf install -y qemu-img rpm coreutils util-linux systemd openssl \
       sed createrepo_c squashfs-tools cdrkit parted e2fsprogs dosfstools \
-      xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify
+      xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify binutils
```

Comment:
You should also update:

- docs/imagecustomizer/how-to/quick-start.md
- toolkit/tools/pkg/imagecustomizerlib/versionsOfToolDependencies.go

Reply:
Thanks @cwize1